README.md

```diff
@@ -1,9 +1,35 @@
 # Joint State Publisher
 
 This contains a package for publishing `sensor_msgs/JointState` messages for a robot described with URDF.
-See the ROS wiki for API documentation and tutorials.
+Given a URDF (either passed on the command-line or via the `/robot_description` topic), this node
+will continually publish default values for all of the movable joints in the URDF to the `/joint_states` topic.
+If the `use_gui` parameter is True, it will also launch a window where the values of the movable joints can be updated.
```

Comment:
As implemented, this line is no longer accurate. If you choose to do the forwarding method from my other comment below, that functionality should be explained here.

More to the point, I would suggest adding something about the two packages, as with https://github.com/ros/urdf.

Also, not that important but, as is, it feels like pulling in these changes to the README doesn't belong in this PR since the changes are unrelated to the separation of the packages. With adding text about the new structure and change to `use_gui`, it would make a lot more sense.

Reply:
That was just a leftover that I forgot to remove.  I've now removed this.